pkg/client_data/lib/publisher_api.dart

```diff
@@ -144,7 +144,7 @@ class InviteMemberRequest {
 
   // json_serializable boiler-plate
   InviteMemberRequest({@required this.reason, @required this.email});
-  factory InviteMemberRequest.fromJson(Map<String, dynamic> json) =>
+  static InviteMemberRequest fromJson(Map<String, dynamic> json) =>
```

Comment:
why?

Reply:
Oh, yeah, nvm..

I suspect `static` is always better since you cannot tear off a constructor, but since I made the thing with code-gen it doesn't matter anymore.